libqtile/widget/wlan.py

```diff
@@ -53,8 +53,10 @@ class Wlan(base.InLoopPollText):
     orientations = base.ORIENTATION_HORIZONTAL
     defaults = [
         ("interface", "wlan0", "The interface to monitor"),
+        ("ethint", "eth0", "The ethernet interface to monitor"),
```

Comment:
I think eth_interface would be a more descriptive name

Reply:
Could you also document that this is only used if there is no wireless connection?

Reply:
Also done!